values.go

```diff
@@ -1521,16 +1521,64 @@ func NewMeteredKeyValuePair(gauge common.MemoryGauge, key, value Value) KeyValue
 
 type Composite interface {
 	Value
-	GetFields() []Field
+
+	isComposite()
+	getFields() []Field
 	getFieldValues() []Value
+
+	SearchFieldByName(fieldName string) Value
+	FieldsMappedByName() map[string]Value
```

Comment:
Feedback from users: Expose existing global functions with the same name as methods on `Composite` for convenience.